web/components/ui/Button.module.css

```
@@ -3,60 +3,108 @@
+    transform: rotate(360deg);
+  }
+}
+
+.srOnly {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.9.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)